src/privileged/blurts/FirefoxMonitor.jsm

```diff
@@ -0,0 +1,766 @@
+        elt.setAttribute("style", "margin-bottom: 1rem;");
+        elt.setAttribute("pack", "center");
+        let imgElt = doc.createElementNS(XUL_NS, "image");
+        imgElt.setAttribute("src", imageDataURIs.iconLaptopRisk);
+        imgElt.setAttribute("style", "width: 148px;");
+        elt.appendChild(imgElt);
+        box.appendChild(elt);
+        elt = doc.createElementNS(XUL_NS, "description");
+        elt.setAttribute("anonid", "maindesc");
+        elt.appendChild(doc.createTextNode("Have an account on this website?\nIt may be at risk."));
+        elt.setAttribute("style", "font-size: 150%; text-align: center; white-space: pre; margin-bottom: 1rem;");
+        box.appendChild(elt);
+        elt = doc.createElementNS(XUL_NS, "description");
+        let strings = [
+          {str: site.Title + " was reported to "},
+          {str: "Firefox Monitor", link: "https://fx-breach-alerts.herokuapp.com/"},
+          {str: ", a service that collects information about data breaches and other ways hackers can steal your information.\n\nEnter your email to find out if your account was included in a data breach. (Note: Your email will not be stored. Find out more in our "},
+          {str: "Privacy Policy", link: "https://www.mozilla.org/privacy/firefox/"},
+          {str: ".)"},
+        ];
+        elt.appendChild(makeSpanWithLinks(strings, doc));
+        elt.setAttribute("style", "text-align: center; white-space: pre-wrap;");
+        box.appendChild(elt);
+        elt = doc.createElementNS(XUL_NS, "textbox");
+        elt.setAttribute("style", "-moz-appearance: none; height: 2.5rem; padding: 0.5rem; background: #FFFFFF; border: 1px solid rgba(12,12,13,0.30); border-radius: 2px;");
+        elt.setAttribute("placeholder", "Enter Email");
+        elt.addEventListener("keydown", function listener(event) {
+          if (event.key !== "Enter") return;
+          elt.removeEventListener("keydown", listener);
+          let popupNotification = doc.getElementById("breach-alerts-notification");
+          doc.getAnonymousElementByAttribute(popupNotification, "anonid", "button").click();
+        });
+        this._textbox = elt;
+        box.appendChild(elt);
+        return box;
+      },
+    };
+    retval.primaryAction = {
+      label: "Search Firefox Monitor",
+      accessKey: "f",
+      callback: function() {
+        FirefoxMonitor.notifyEventListeners(`variant_2_submit`);
+        let stringStream = Cc["@mozilla.org/io/string-input-stream;1"].
+          createInstance(Ci.nsIStringInputStream);
+        stringStream.data = `emailHash=${sha1(this._textbox.value)}`;
+
+        let postData = Cc["@mozilla.org/network/mime-input-stream;1"].
+          createInstance(Ci.nsIMIMEInputStream);
+        postData.addHeader("Content-Type", "application/x-www-form-urlencoded");
+        postData.setData(stringStream);
+        doc.defaultView.openUILinkIn("https://fx-breach-alerts.herokuapp.com/scan", "tab", { postData });
```

Comment:
Per #6; "Change links to server to monitor.firefox.com"
We'll need to confirm the production endpoint with @micheletto, and make sure the add-on and all server bits are pointing to production endpoints and not our Heroku dev server.

Reply:
> Per #6; "Change links to server to monitor.firefox.com"
> We'll need to confirm the production endpoint with @Micheletto, and make sure the add-on and all server bits are pointing to production endpoints and not our Heroku dev server.

That's the correct endpoint, but I would prefer erroring out on missing configuration options over hardcoding any defaults.